pkg/build/nodeimage/build.go

```diff
@@ -17,7 +17,13 @@ limitations under the License.
 package nodeimage
 
 import (
+	"fmt"
+	"os"
+	"regexp"
 	"runtime"
+	"strconv"
+
+	"github.com/blang/semver/v4"
```

Comment:
@BenTheElder forked it from apimachinery to avoid dependencies `pkg/internal/version/version.go`